frontend/src/views/Org/MembersPage/MembersPage.tsx

```diff
@@ -1,62 +1,48 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
-import { useTranslation } from "react-i18next";
-import { motion } from "framer-motion";
-
 import { Tab, TabList, TabPanel, Tabs } from "@app/components/v2";
-import { OrgPermissionActions, OrgPermissionSubjects, useOrganization } from "@app/context";
+import { OrgPermissionActions, OrgPermissionSubjects } from "@app/context";
 import { withPermission } from "@app/hoc";
-import { useGetRoles } from "@app/hooks/api";
-import { TRole } from "@app/hooks/api/roles/types";
 
-import { OrgMembersTable } from "./components/OrgMembersTable";
-import { OrgRoleTabSection } from "./components/OrgRoleTabSection";
+import {
+  OrgMachineIdentityTab,
+  OrgMembersTab,
+  OrgRoleTabSection} from "./components";
 
 enum TabSections {
   Member = "members",
-  Roles = "roles"
+  Roles = "roles",
+  MachineIdentities = "machine-identities"
 }
 
 export const MembersPage = withPermission(
   () => {
-    const { t } = useTranslation();
-    const { currentOrg } = useOrganization();
-
-    const orgId = currentOrg?._id || "";
-
-    const { data: roles, isLoading: isRolesLoading } = useGetRoles({
-      orgId
-    });
-
     return (
       <div className="container mx-auto flex flex-col justify-between bg-bunker-800 text-white">
         <div className="mb-6 w-full py-6 px-6 max-w-7xl mx-auto">
           <p className="mr-4 mb-4 text-3xl font-semibold text-white">
-            {t("section.members.org-members")}
+            Access Control
```

Comment:
The members `Access Control` page looks very similar to org Access Control page and i forget what page i am on often. What if we rename to `Organization Cccess Control`?

Reply:
We can do that